bpf/bpf_netdev.c

```diff
@@ -324,6 +349,13 @@ static __always_inline int handle_ipv4(struct __ctx_buff *ctx,
 		if (ret < 0)
 			return ret;
 	}
+
+#if defined(NO_REDIRECT)
```

Comment:
We should be able to get rid of the `NO_REDIRECT` once the lxc outbound path redirects instead of returning `TC_ACT_OK`.

Reply:
I think the worry is that the fib lookup may fail still.